charts/radar-self-enrolment-ui/README.md

```diff
@@ -1,8 +1,6 @@
-
-
 # radar-self-enrolment-ui
 
-![Version: 0.2.1](https://img.shields.io/badge/Version-0.2.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: dev](https://img.shields.io/badge/AppVersion-dev-informational?style=flat-square)
+![Version: 0.2.2](https://img.shields.io/badge/Version-0.2.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: dev](https://img.shields.io/badge/AppVersion-dev-informational?style=flat-square)
```

Comment:
Does the image version need to be updated from `dev` to a specific version?

Reply:
Updated now to `0.0.1` 👍